### PR TITLE
added extra methods to the product

### PR DESCRIPTION
**Pull request by hosseintoussi**

### CODE REVIEW

**Comment by zfhui** [comment]
LGTM, did you check, if there any tests to add for this?

**Comment by hosseintoussi** [comment]
i didnt find much tests on this but let me add them.

File: lib/onix/onix21.rb [hunk 1]
```diff
@@ -45,6 +45,13 @@ def self.code_ident
       end
     end
 
+    class ProductForm < CodeFromYaml
+      private
+      def self.code_ident
```
Comment (zfhui) [inline lib/onix/onix21.rb]: This is also not working. I just tested, `self.code_ident` doesn't get registered as a private class method.

Comment (hosseintoussi) [reply lib/onix/onix21.rb]: I am just keeping this the same style of the other types defined. I can remove the `private`.

Comment (dinks) [reply lib/onix/onix21.rb]: I think to make them really private we need `class << self`

Comment (hosseintoussi) [reply lib/onix/onix21.rb]: or you can also pass them to `private_class_method` :)

**Pull request merged**